reference/tax_settings.v3.yml

```diff
@@ -0,0 +1,147 @@
+openapi: 3.0.0
+x-stoplight:
+  id: 68ccfa5c69bbb
+info:
+  title: Tax Settings
+  version: ''
+  description: Manage tax settings
+servers:
+  - url: 'https://api.bigcommerce.com'
+paths:
+  '/stores/{store_hash}/v3/tax/settings':
+    get:
+      tags:
+        - Tax Settings
+      summary: Get Tax Settings
+      description: Retrieves global-level tax settings.
+      operationId: get-tax-settings
+      parameters:
+        - $ref: '#/components/parameters/storehash'
+      responses:
+        '200':
+          description: OK
+          content:
+            '*/*':
```

Comment:
```suggestion
            '*/*': Do we need this?
```

Reply:
i think it was a placeholder for content type, so i changed it to "application/json"